scripts/templates/2.0/mapping.json

```diff
@@ -363,6 +363,51 @@
                 "section": "cloud_verifier",
                 "option": "log_destination",
                 "default": ""
+            },
+            "durable_attestation_import" : {
```

Comment:
@THS-on @ansasaki maybe you can see if the new files are following the naming spirit after the refactoring? I am very bad for this kind of things.

Reply:
@aplanas Sorry for taking long to reply, I'll check

Reply:
@ansasaki I think it is fine for now because we merge it as an experimental feature.